initial/pom.xml

```diff
@@ -15,6 +15,16 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.springframework.cloud</groupId>
```

Comment:
I guess that points to a key thing that's missing in the guide, which is the instruction to add the `spring-cloud-gcp-starter-pubsub` dependency to the app. @joaoandremartins